### DISCUSSION TITLE
How to deal with ordinal features?

### DISCUSSION THREAD

**Discussion by anakin87** [ludwig-ai/ludwig#3374] [Q&A] [answered]
Hello... Great framework, as always!

I am working with a dataset for fake news classification.
The label can take a value from 0 (certainly fake) to 3 (certainly true).

How can I model the label using Ludwig?
If I use the type "category", I lose the ordinal information. (In fact, I get worse results than using a logistic regression)
The "number" type doesn't seem to work for me.

Any advice?

**Comment by tgaddair** [comment, marked as answer]
Hey @anakin87, great question! One option would typically be to use the NUMBER type and then post-process the output to conform to the correct set of outputs.

That said, there's been some interesting work recently with adding ordinal loss functions for multi-class classification. I took a stab at adding one of the newer ones (CORN loss) to Ludwig in #3375. Please try it out and let me know if it works better for your task.

**Reply by anakin87** [reply]
Thank you @tgaddair. Great insights!
I will try CORN...
